src/styles/burger.scss

```
@@ -0,0 +1,53 @@
+.burger {
+  display: block;
+  position: relative;
+  width: 30px;
+  height: 20px;
+  z-index: 3;
+
+  &__line {
+    position: absolute;
+    background-color: $font-color;
+    left: -10px;
+    width: 100%;
+    height: 2px;
+    top: 9px;
+    transition: all 0.2s linear;
+  }
+  &::before,
+  &::after {
+    content: "";
+    background-color: $font-color;
+    position: absolute;
+    width: 100%;
+    height: 2px;
+    left: -10px;
+    transition: all 0.2s linear;
```

Явно вказуй які властивості анімуєш
```suggestion
    transition: all 0.2s linear;
```